crates/store/re_chunk_store/src/gc.rs

```diff
@@ -683,10 +691,9 @@ impl ChunkStore {
 
         {
             re_tracing::profile_scope!("retain");
-            self.chunk_ids_per_min_row_id.retain(|_row_id, chunk_ids| {
-                chunk_ids.retain(|chunk_id| !chunk_ids_removed.contains(chunk_id));
-                !chunk_ids.is_empty()
-            });
+            for row_id in row_ids_removed {
+                self.chunk_ids_per_min_row_id.remove(&row_id);
```

Comment:
Note: this is technically incorrect because it doesn't account the vector within `chunk_ids_per_min_row_id`, except that vector doesn't make any sense. I'm going to remove it entirely tomorrow: going out of our way to pseudo-support undefined behavior (duplicated Row IDs) is unproductive at best, non-sensical at worst.